Source/Chronozoom.UI/ui/auth-edit-timeline-form.html

```diff
@@ -30,13 +30,32 @@
     <div class="cz-form-time-end" style="display: block"></div>
 
     <br />
-    <label class="cz-lightgray cz-form-label">End Time *</label>
-    <div class="cz-form-time-end" style="display: block"></div>
+    <label class="cz-lightgray cz-form-label cz-form-offset-label">
+        Top boundary (<span title="Distance to child timeline top measured in percents of parent timeline height" class="cz-form-percent-tooltip">%</span>)
+    </label>
+    <div class="cz-input-wrapper" style="display:inline-block; width:10%">
+        <input class="cz-form-item-top-bound-offset cz-form-item-offset cz-input" type="text" maxlength="200">
+    </div>
+
+    <br />
+    <label class="cz-lightgray cz-form-label cz-form-offset-label">
+        Bottom boundary (<span title="Distance to child timeline bottom measured in percents of parent timeline height" class="cz-form-percent-tooltip">%</span>)
+    </label>
+    <div class="cz-input-wrapper" style="display:inline-block; width:10%">
+        <input class="cz-form-item-bottom-bound-offset cz-form-item-offset cz-input" type="text" maxlength="200">
+    </div>
 
     <br />
+    <br />
     <label class="cz-lightgray cz-form-label">Import background image from</label>
-    <div class="cz-form-medialist"></div>
- 
+    <div class="cz-form-medialist cz-medialist">
+        <div class="cz-medialist-item" title="bing" media-picker="bing">
```

Comment:
These items are being generated in JS counterpart during load, no need to add them here